src/partitioning/block.jl

```diff
@@ -55,16 +57,28 @@ function partition(object, method::BlockPartition)
     append!(subsets[i], j)
   end
 
+  if calculate_metadata == false
+    println("Metadata calculation has been disabled. To calculate Metadata, please enable it while calling BlockPartition with neighbors = true.") 
+  end
```

Comment:
Remove any print statement. Print statements usually appear in end user code but you are contributing to a library that people will use in high-level applications where they can print whatever they want. Makes sense?

Reply:
Yep! Fixed it!